Makefile

```diff
@@ -21,7 +21,8 @@ AUTO_GENERATED = node_exporter \
 mysqld_exporter \
 redis_exporter \
 haproxy_exporter \
-postgres_exporter
+postgres_exporter \
+nginx-prometheus-exporter
```

Comment:
Could you please rename the package to `nginx_exporter` to be consistent with names of other packages?